confidant/authnz/__init__.py

```diff
@@ -96,8 +74,8 @@ def check_csrf_token():
     # csrf tokens.
     if g.auth_type == 'kms':
         return True
-    token = request.headers.get('X-XSRF-TOKEN')
-    return safe_str_cmp(token, session.get('XSRF-TOKEN'))
+    token = request.headers.get('X-XSRF-TOKEN', '')
```

Comment:
I wonder if we should check to see if token is None and return False here if so.